README.md

```diff
@@ -50,6 +50,28 @@ have dependencies on other algebras which must be implemented.
     - Two promises are equivalent when they yield equivalent values.
     - Two functions are equivalent if they yield equivalent outputs for equivalent inputs.
 
+## Type signature notation
+
+The type signature notation used in this document is described [here](https://sanctuary.js.org/#types). In summary:
+
+1. New types can be created via type constructors. Type constructors take one or more type arguments. For example, the type "array of strings" is written as `Array String`, and the type constructor `Array` can be defined as `Array :: a -> a`.
```

Comment:
There really is a `Type` type: sanctuary-js/sanctuary-def#125.

If we're talking the [`$.Array`][1] type constructor, it is not polymorphic – it must be applied to a `Type` value such as [`$.Integer`][2]. `Array :: a -> a` would be an oddly named identity function. ;)


[1]: https://github.com/sanctuary-js/sanctuary-def#Array
[2]: https://github.com/sanctuary-js/sanctuary-def#Integer